.storybook/story-wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+import "./webapp-styles.less";
+
+/**
+ * Wrapper for Storybook stories that
+ * 1. Renders the story inside of a .framework-perseus container
+ * 2. Includes the global styles from webapp
```

Comment:
I would love it if we could isolate Perseus from these types of resets that trickle down into Perseus from outside. It makes sense to try to align styling with that of our primary consumer (webapp), but, if possible, could we make these styles/resets explicit in our Perseus styling system instead of mimicking them in Storybook?